people/_posts/0000-04-01-alaa-taha

```
@@ -0,0 +1,21 @@
+---
+name: Alaa Taha
+degree: 
+position: Master's student 
+affiliation: Biomedical Engineering 
+cosupervisor: Dr. Jonathan Lau 
+image: /assets/images/people/alaa-taha.jpg 
```

This file was not added - will default to the placeholder in the meantime!